osu.Framework/Graphics/Containers/DrawSizePreservingFillContainer.cs

```diff
@@ -102,4 +136,13 @@ public enum DrawSizePreservationStrategy
         /// </summary>
         Separate,
     }
+
+    public class MustFillToParentException : InvalidOperationException
+    {
+        public MustFillToParentException()
+            : base($"This must fill up to the parent's determined {nameof(CompositeDrawable.ChildSize)}, "
+                   + "please modify the size of the parenting container instead.")
```

Comment:
this doesn't really tell a consumer all that well what is going on

```suggestion
            : base($"A {nameof(DrawSizePreservingFillContainer)} will automatically fill its parent's {nameof(CompositeDrawable.ChildSize)}). "
                   + "Modifying its dimensions is not supported.")
```

Reply:
Classic english issues from me :P

Though slight change in your documentation, shouldn't 
 - "A DrawSizePreservingFillContainer will automatically **fill** its parent's ChildSize" 

become 
 - "A DrawSizePreservingFillContainer will automatically **fill up to** its parent's **determined** ChildSize"?

Reply:
Either's clear enough to me I suppose.